src/api.ts

```diff
@@ -274,6 +275,14 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
+  upload(selector: string, files: string[] | string): Chromeless<T> {
```

Comment:
I think the name `upload` is a bit misleading since all this is doing is selecting files in a file input. Maybe rather use `selectFile` instead?